driver-core/src/main/com/mongodb/internal/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -44,7 +48,12 @@
     private final int maxWireVersion;
 
     private volatile BsonDocument resumeToken;
-    private volatile AsyncAggregateResponseBatchCursor<RawBsonDocument> wrapped;
+    /**
+     * {@code null} {@linkplain AtomicReference#get() value} combined with {@link #isClosed} being {@code false}
+     * represents a situation in which the wrapped object was closed by {@code this} and {@code this} stayed open.
+     * This happens as a result of {@linkplain ChangeStreamBatchCursorHelper#isRetryableError(Throwable, int) retryable errors}.
```

Comment:
TIL about labels for linkplain

Reply:
I found the text hard to understand. Perhaps simplify to something like:

```
  /**
   * Note: {@linkplain ChangeStreamBatchCursorHelper#isRetryableError(Throwable, int) Retryable errors} can 
   * result in {@code wrapped} containing a {@code null} value and {@link #isClosed} being {@code false}. 
   * This represents a situation in which the wrapped object was closed by {@code this} but {@code this} remained open.
```

Reply:
Ross, thank you for providing the proposed text. Used it as suggested.